packages/legend-query/src/components/QueryBuilderMilestoneEditor.tsx

```diff
@@ -52,6 +51,7 @@ import {
 import { Dialog, CustomSelectorInput, RefreshIcon } from '@finos/legend-art';
 import format from 'date-fns/format/index';
 import { addDays } from 'date-fns';
+import { setGenericTypeRawType } from '@finos/legend-studio';
```

Comment:
Let's create yet another set of method for `legend-query`. We should not have any depdencies on `legend-studio` from `Legend Query`.